src/main/resources/static/js/survey.js

```diff
@@ -115,6 +119,11 @@ function createSurvey() {
         // The class of the element is used to identify the type of question
         if ($(element).hasClass('openEnded')) {
             survey.questions.push({ type: "openEnded", question: question })
+        } else if($(element).hasClass('numberQuestion')) {
```

Comment:
min and max should be converted to numbers. Try parseInt().

Reply:
@rinadina was this tested? I think it will crash right now since it doesn't cast to an int. Nice work using `let` by the way

Reply:
For the line below this

Reply:
Added parseInt